src/app/containers/AudioVideo/audioVideoMetadata.js

```diff
@@ -19,16 +17,19 @@ const videoMetadata = blocks => {
     block => block.type === 'aresMediaMetadata',
   );
 
-  aresMetaDataBlocks.forEach(block =>
+  aresMetaDataBlocks.forEach(block => {
+    const format = deepGet(['model', 'format'], block);
+    const type = format === 'audio' ? 'AudioObject' : 'VideoObject';
+
     listContent.push({
-      '@type': videoObject,
+      '@type': type,
       name: deepGet(['model', 'title'], block),
       description: deepGet(['model', 'synopses', 'short'], block),
       duration: deepGet(['model', 'versions', [0], 'duration'], block),
       thumbnailUrl: `https://${deepGet(['model', 'imageUrl'], block)}`,
```

Comment:
This isn't the correct format.

This will return `https://ichef.test.bbci.co.uk/images/ic/$recipe/p01mxrbn.jpg` which is not a valid image URL. We need to define sizes.

The spec suggests using an array of images of different sizes so that:
> Google will pick the best image to display in Search results based on the aspect ratio and resolution.

https://developers.google.com/search/docs/data-types/video#type_definitions

Reply:
To be fixed in https://github.com/bbc/simorgh/issues/2413